.github/workflows/ios-build-test-fabric.yml

```diff
@@ -0,0 +1,37 @@
+name: Test iOS build on Fabric
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'ios/**'
```

Comment:
```suggestion
      - 'ios/**'
      - 'common/**'
      - 'src/fabric/**'
      - 'FabricExample/**'
      - 'package.json'
      - 'RNScreens.podspec'
```

Reply:
We would need to add `package.json` and `RNScreens.podspec` because they can affect codegen. 
What do you think about also adding `FabricExample` here?